packages/round-manager/src/features/api/contracts.ts

```diff
@@ -121,13 +123,48 @@ const qfVotingStrategyFactoryMap: ChainIdToStringMap = {
   [ChainId.POLYGON_MUMBAI]: "0xF7c101A95Ea4cBD5DA0Ab9827D7B2C9857440143",
 };
 /* QuadraticFundingVotingStrategy */
-export const qfVotingStrategyFactoryContract = (chainId: ChainId): Contract => {
+export const qfVotingStrategyFactoryContract = (
+  chainId: ChainId
+): Contract & { address: string } => {
   return {
     address: qfVotingStrategyFactoryMap[chainId],
     abi: abi.qfVotingStrategyFactory,
   };
 };
 
+/* VotingStrategyFactory */
+export const dgVotingStrategyDummyContract = (
+  chainId: ChainId | undefined
```

Comment:
it doesn't make sense for the chainId argument to be undefined here, it's most likely just a downstream effect of bad optional management.